aim/sdk/metric.py

```diff
@@ -252,114 +198,148 @@ def dataframe(
         include_context=True
     ) -> 'DataFrame':
         dfs = [
-            trace.dataframe(include_run=include_run,
-                            include_name=include_name,
-                            include_context=include_context,
-                            only_last=only_last)
-            for trace in self
+            metric.dataframe(include_run=include_run,
+                             include_name=include_name,
+                             include_context=include_context,
+                             only_last=only_last)
+            for metric in self
         ]
         if not dfs:
             return None
         import pandas as pd
         return pd.concat(dfs)
 
-    def __iter__(self) -> Iterator[Trace]:
+    def __iter__(self) -> Iterator[Metric]:
         return self.iter()
 
     @abstractmethod
-    def iter(self) -> Iterator[Trace]:
-        ...
-
-    def iter_series(self) -> Iterator[Trace]:
-        # Alias to iter
-        return self.iter()
+    def iter(self) -> Iterator[Metric]:
+        """Get iterator for collection metrics
```

Comment:
maybe collection of metrics or metrics collection? (in some other places as well the same issue is present)